### PR TITLE
change env to gradle properties

### PR DESCRIPTION
**Pull request by sergioboi**
Save ~ 2min of build

### CODE REVIEW

File: _main.sh [hunk 1]
```diff
@@ -87,15 +95,7 @@ function runInBuilder() {
     --workdir /app \
     --env GRADLE_USER_HOME=/gradle \
     --env LOCAL_USER_ID="$USER_ID" \
-    --env BINTRAY_USER="$BINTRAY_USER" \
-    --env BINTRAY_API_KEY="$BINTRAY_API_KEY" \
     --env BINTRAY_GPG_PASSPHRASE="$BINTRAY_GPG_PASSPHRASE" \
```
Comment (sergioboi) [inline _main.sh:98]: Is env used by bintray plugin? I decided to keep it because it doesn't influence our configuration

File: build.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ TEMP_PROJECT_VERSION="temp-version"
 
 # `tasks` triggers full tasks graph resolving, checking for possible misconfigurations
 runInBuilder "./gradlew tasks build publishToMavenLocal ${GRADLE_ARGS} -PprojectVersion=${TEMP_PROJECT_VERSION};
-    ./gradlew :subprojects:android-test:test-app:instrumentationUi ${GRADLE_ARGS} -PinfraVersion=${TEMP_PROJECT_VERSION}"
+    ./gradlew :subprojects:android-test:test-app:instrumentationUi ${GRADLE_ARGS} -PinfraVersion=${TEMP_PROJECT_VERSION};
+    ./gradlew help"
```
Comment (sergioboi) [inline build.sh]: run without GRADLE_ARGS it the same as run new docker without env

File: buildSrc/src/main/java/Dependencies.kt [hunk 3]
```diff
@@ -55,6 +55,9 @@ object Dependencies {
     object gradle {
         val androidPlugin = "com.android.tools.build:gradle:3.5.3"
         val kotlinPlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlin}"
+        object avito {
+            val kotlinDslSupport = "com.avito.android:kotlin-dsl-support"
```
Comment (sergioboi) [inline buildSrc/src/main/java/Dependencies.kt:59]: Minor change. Need to use in buildscript. I don't know how use infraVersion in buildSrc

File: build.sh (hunk 2 above)
Comment (eugene-krivobokov) [reply build.sh]: What about running it as the first command?

```
runInBuilder " 
1 mavenLocal
2 help
3 instrumentationUI
"
```

It shows problems in configuration. It's easier to catch them as early as possible, in the simplest run.

File: subprojects/gradle/slack/build.gradle.kts [hunk 4]
```diff
@@ -21,3 +30,16 @@ dependencies {
...
+
+tasks.withType(Test::class.java).forEach { testTask ->
+    with(testTask) {
+        val testProperties = listOf(
+            "avito.slack.test.channel",
+            "avito.slack.test.token",
+            "avito.slack.test.workspace"
+        )
+        testProperties.forEach { key ->
+            systemProperty(key, project.getOptionalStringProperty(key) ?: "")
```
Comment (eugene-krivobokov) [inline subprojects/gradle/slack/build.gradle.kts:42]: Good idea to have the same name of env and gradle property

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**